controllers/activemqartemis_controller.go

```diff
@@ -121,7 +121,7 @@ type ActiveMQArtemisReconciler struct {
 //+kubebuilder:rbac:groups=monitoring.coreos.com,namespace=activemq-artemis-operator,resources=servicemonitors,verbs=get;create
 //+kubebuilder:rbac:groups=apps,namespace=activemq-artemis-operator,resources=deployments/finalizers,verbs=update
 //+kubebuilder:rbac:groups=rbac.authorization.k8s.io,namespace=activemq-artemis-operator,resources=roles;rolebindings,verbs=create;get;delete
-//+kubebuilder:rbac:groups=policy,namespace=activemq-artemis-operator,resources=poddisruptionbudgets,verbs=create;get;delete
+//+kubebuilder:rbac:groups=policy,namespace=activemq-artemis-operator,resources=poddisruptionbudgets,verbs=create;get;delete;list;update;watch
```

Comment:
this watch permission. I don't think that is necessary. Is it? we should not watch it b/c we already watch the CR.

Reply:
this watch permission is needed, but not directly. If the permission is absent, the operator got issues with client cache updates. (i.e. when you create the resource the cache won't get updated quickly). (same thing with other resources like services which we don't directly watch them).